openupgrade_scripts/scripts/hr_holidays/18.0.1.6/pre-migration.py

```diff
@@ -51,7 +51,7 @@ def update_states(env):
         env.cr,
         """
         UPDATE hr_leave_allocation
-        SET state = 'cancel'
+        SET state = 'refuse'
```

Comment:
why?

Reply:
Cancel state was only added to hr_leave but does not exist on hr_leave_allocation